wait/host.go

```diff
@@ -74,9 +75,9 @@ func UntilMasterUserRecordHasUserAccountStatuses(expUaStatuses ...toolchainv1alp
 			return false
 		}
 		for _, expUaStatus := range expUaStatuses {
-			expUaStatus.SyncIndex = getUaSpecSyncIndex(mur, expUaStatus.TargetCluster)
+			expUaStatus.SyncIndex = getUaSpecSyncIndex(mur, expUaStatus.Cluster.Name)
 			if !containsUserAccountStatus(mur.Status.UserAccounts, expUaStatus) {
-				a.T.Logf("waiting for UserAccount status to be present in MasterUserRecord '%s`", mur.Name)
+				a.T.Logf("waiting for UserAccount status to be present in MasterUserRecord '%s'. Actual status: '%v'; Expected status: '%v'", mur.Name, mur.Status.UserAccounts, expUaStatus)
```

Comment:
```suggestion
				a.T.Logf("waiting for UserAccount status to be present in MasterUserRecord '%s'. Actual status: '%v'; Expected statuses: '%v'", mur.Name, mur.Status.UserAccounts, expUaStatus)
```

Reply:
Re-worded it a bit - https://github.com/codeready-toolchain/toolchain-e2e/pull/51/commits/fa602c224831eff6e45ae93f94aba7670a238af7